PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -198,15 +202,24 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
         electronVeto = Var("passElectronVeto()",bool,doc="pass electron veto"),
         pixelSeed = Var("hasPixelSeed()",bool,doc="has pixel seed"),
         mvaID = Var("userFloat('mvaID')",float,doc="MVA ID score, Fall17V2",precision=10),
-        mvaID_Fall17V1p1 = Var("userFloat('mvaID_Fall17V1p1')",float,doc="MVA ID score, Fall17V1p1",precision=10),
         mvaID_WP90 = Var("userInt('mvaID_WP90')",bool,doc="MVA ID WP90, Fall17V2"),
         mvaID_WP80 = Var("userInt('mvaID_WP80')",bool,doc="MVA ID WP80, Fall17V2"),
+        pfPhoIso03 = Var("photonIso()",float,doc="PF absolute isolation dR=0.3, photon component (uncorrected)"),
+        pfChargedIsoPFPV = Var("chargedHadronPFPVIso()",float,doc="PF absolute isolation dR=0.3, charged component (PF PV only)"),
+        pfChargedIsoWorstVtx = Var("chargedHadronWorstVtxIso()",float,doc="PF absolute isolation dR=0.3, charged component (Vertex with largest isolation)"),
         pfRelIso03_chg = Var("userFloat('PFIsoChg')/pt",float,doc="PF relative isolation dR=0.3, charged component (with rho*EA PU corrections)"),
         pfRelIso03_all = Var("userFloat('PFIsoAll')/pt",float,doc="PF relative isolation dR=0.3, total (with rho*EA PU corrections)"),
         hoe = Var("hadronicOverEm()",float,doc="H over E",precision=8),
```

Comment:
hoe = Var("hadronicOverEm()",float,doc="H over E",precision=8),
based on the discussion at the xPOG this is neither the run2 definition (i.e. what goes in the ID is another one) nor the one planned for Run3.

We should update with the right one.

Reply:
In run 2 the H/E variable used in IDs is indeed hadTowOverEm() (tower based H/E), while for Run 3 we are planning to move to the cone based H/E, ie hadronicOverEm() (the one you are talking about), so probably we should keep it?

Reply:
we can have both Run2 and Run3 right
```
(run2_egamma_2016 || run2_egamma_2017 || run2_egamma_2018) .toModify(photonTable,
    hoe = Var("hadTowOverEm()",float,doc="H over E (Run2)",precision=8),
)
```

Reply:
does this duality hadTowOverEm() vs hadronicOverEm() exist also for the electrons ?
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/electrons_cff.py#L388

Reply:
No, for electrons we do not need hadTowOverEm(): the ID cuts on hadronicOverEm(), which is the one included